Android/app/src/main/res/values/strings.xml

```diff
@@ -104,8 +105,12 @@
     <string name="share_intent_text">Share Contact Details</string>
     <string name="share_menu">Share</string>
     <string name="share_contact_text">Share Contact</string>
+    <string name="share_contact">Share My Contact</string>
```

Comment:
these strings also.

Reply:
In some old Toast messages the message String was typed directly in the makeText() so I replaced those with String rescources instead. You liked consistency right? 😃

Reply:
But you haven't used this string anywhere